docs/reuse/tutorial/setup_edge.rst

```diff
@@ -74,8 +75,9 @@ If successful, the terminal will output ``microk8s is running``
 along with a list of enabled and disabled add-ons.
 
 Juju is required to deploy the |12FactorApp| application.
-Install Juju using the ``3.6/stable`` track, and bootstrap a
-development controller:
+We'll install Juju using the ``3.6/stable`` track; since the snap is
+strictly confined, we'll also manually create a directory to contain
```

Comment:
> strictly confined

Makes perfect sense to me, but maybe not to generalists who lack snap knowledge. How about "sandboxed"? I think that's more universally understood.